pycbc/io/record.py

```diff
@@ -324,6 +324,14 @@ def merge_arrays(merge_list, names=None, flatten=True, outtype=None):
         A new array with all of the fields in all of the arrays merged into
         a single array.
     """
+    # make sure everything in merge_list is an array
+    for ii,arr in enumerate(merge_list):
```

Comment:
This and the lines below seem to be identical. Can you make this a function instead and simply call it both places. Also why not use the more explicit numpy.array(data, ndmin=1) to ensure a 1 dimensional shape.  No checking is then needed on the shape, and it can all be done in a single line as in 330.